.github/dependabot.yml

```diff
@@ -4,12 +4,39 @@ updates:
     directory: /
     schedule:
       interval: monthly
+    groups:
+      github-actions:
+        patterns:
+          - "*"
   - package-ecosystem: docker
     directory: /
     schedule:
       interval: monthly
+    groups:
+      docker:
+        patterns:
+          - "*"
   - package-ecosystem: gomod
     open-pull-requests-limit: 10
     directory: /
     schedule:
       interval: monthly
+    groups:
+      aws:
+        patterns:
+          - "github.com/aws/*"
+      docker:
+        patterns:
+          - "github.com/docker/*"
+          - "github.com/moby/*"
+      testcontainers:
+        patterns:
+          - "github.com/testcontainers/*"
+      common:
```

Comment:
I thought about adding some kind of prefix (like `gomod-common`, `gomod-docker`, etc.) for clearer PR titles.
`chore(deps): add 2 updates to the testcontainers group` -> `chore(deps): add 2 updates to the gomod-testcontainers group.
But `dependabot` adds labels (`go`, `github_action`, etc.).
So maybe this isn't necessary.

@knqyf263 wdyt?

Reply:
Yes, the label looks good enough.